apps/web/pages/api/book/recurring-event.ts

```diff
@@ -49,11 +51,14 @@ async function handler(req: NextApiRequest & { userId?: number }, res: NextApiRe
 
     const eachRecurringBooking = await handleNewBooking(recurringEventReq, {
       isNotAnApiCall: true,
+      skipAvailabilityCheck: key >= numSlotsToCheckForAvailability,
```

Comment:
just to make sure, the skipAvailabilityCheck will be false for 3 iterations of the loop, it's a bit misleading with the name and value of numSlotsToCheckForAvailability. is it wanted ?

Reply:
No, I think there is some confusion. 

0>=2 & 1>=2 are the only values for which it would be false
2>=2 and 3>=2...... and all other values will have truthy value.

Reply:
haha yeah my bad I read this too fast